### ISSUE TITLE
nnunet 多卡支持问题

### ISSUE THREAD

**Issue by HongweiChe** [PaddlePaddle/PaddleSeg#3272]
nnuet多卡支持， 添加代码：
sampler = DistributedBatchSampler(train_dataset,
                                        batch_size=batch_size,
                                        shuffle=False,
                                        drop_last=False)
        loader = DataLoader(train_dataset,
                                batch_sampler=sampler,
                                num_workers=num_workers,
                                return_list=True,
                                worker_init_fn=worker_init_fn,)
存在
 we only support 'to_tensor()' in dynamic graph mode, please call 'paddle.disable_static()' to enter dynamic graph mode.    
在动转静情况下，我们怎么操作才能支持多卡？

请问nnunet支持多卡上有什么困难或是问题吗？为什么其他模型支持多卡仅nnunet不支持多卡？

**Comment by shiyutang** [comment]
为什么动转静需要多卡呢？ 另外你的报错显示代码中存在了动转静不支持的to_tensor操作。
